plugins/decl_hdf5/README.md

```diff
@@ -145,6 +145,8 @@ The possible values for the keys are as follow:
   See
   https://support.hdfgroup.org/HDF5/doc/RM/RM_H5P.html#Property-SetFletcher32
   for more information.
+* `mpio` : a string expression to define the type of MPIIO parallel pointer 
```

Comment:
It is not better to write mpiio instead of mpio?

Reply:
In hdf5, they used  MPIO  instead of  MPIIO (For example "H5FD_MPIO_COLLECTIVE"). Perhaps, it is better to use MPIO to be coherent with HDF5.